.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
 - pip install -U pip
 - pip install -U pytest pytest-cov
 - pip install codecov
+- sudo apt-get install lilypond
```

Comment:
Don't you also need to install music21 as well?

Reply:
music21 is a python package; will be installed within requirement.txt